server/pbench/lib/config/pbench-server.cfg.example

```diff
@@ -11,6 +11,13 @@ pbench-local-dir = /srv/pbench-local
 # Backup dir may be remote NFS mounted
 pbench-backup-dir = /mnt/pbench.archive.backup
 
+[aws-s3]
+endpoint_url =
+aws_access_key_id =
+aws_secret_access_key =
+bucket_name = 
+target_dir = /pbench/archive/fs-version-001
```

Comment:
Can we drop this parameter and just use the existing `pbench-archive-dir` configuration parameter?

Reply:
Done